.travis/install_conda_osx.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Download Miniconda installer
+wget 'https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh' -O 'miniconda.sh'
```

Comment:
Since this is the only part of the script that changes from the Linux version could we just make the installer name a parameter and modify the calls in `.travis.yml`?

```suggestion
wget "https://repo.continuum.io/miniconda/$1" -O 'miniconda.sh'
```

Reply:
Agreed on both. Thanks!